init/action.yml

```
@@ -23,6 +23,9 @@ inputs:
     description: Try to auto-install your python dependencies
     required: true
     default: 'true'
+outputs:
+  codeql-path:
+      description: The path of the CodeQL binary used for analysis
```

```suggestion
    description: The path of the CodeQL binary used for analysis
```

Just to make the indentation consistent with the rest of the file.

---

Oh, nice catch! 👍 Need to set up vscode better 🤔